extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/src/main/java/org/eclipse/smarthome/binding/bosesoundtouch/internal/XMLResponseHandler.java

```diff
@@ -492,6 +499,14 @@ public void characters(char[] ch, int start, int length) throws SAXException {
                 break;
             case InfoType:
                 setConfigOption(DEVICE_INFO_TYPE, new String(ch, start, length));
+                setConfigOption(Thing.PROPERTY_MODEL_ID, new String(ch, start, length));
+                break;
+            case InfoModuleType:
+                setConfigOption(Thing.PROPERTY_HARDWARE_VERSION, new String(ch, start, length));
+                break;
+            case InfoFirmwareVersion:
+                String[] fwVersion = new String(ch, start, length).split(" ");

```

Comment:
And that applies here as well :-)

Reply:
I think here isn't required as `ch` is already a `char[]`

Reply:
Indeed 👍